napari/layers/image/image.py

```diff
@@ -527,18 +534,46 @@ def interpolation(self):
         str
             The current interpolation mode
         """
-        return str(self._interpolation[self._ndisplay])
+        return str(
+            self._interpolation2d
+            if self._ndisplay == 2
+            else self._interpolation3d
+        )
```

Comment:
Either this should be deprecated too, or we should keep the setter. I'm fine either way, but maybe it would be less confusing to simply remove it.

Reply:
Not sure if you added/updated the setter since this comment, but if the setter is deprecated (and it probably should be), my opinion is that the getter should be too.

Reply:
done